folding-schemes/src/folding/nova/mod.rs

```diff
@@ -93,6 +97,84 @@ where
     }
 }
 
+// The out-circuit counterpart of `nonnative_field_var_to_constraint_field` in `nova/cyclefold.rs`
+fn nonnative_field_to_field_elements<TargetField: PrimeField, BaseField: PrimeField>(
```

Comment:
As in the prev comment, how would you see to move this method in the `folding-schemes/src/folding/circuits/*` dir?